.goreleaser.yml

```diff
@@ -1,5 +1,3 @@
-env:
-  - GO111MODULE=on
```

Comment:
`GO111MODULE` is enabled by default from Go1.16.

Ref: https://golang.org/doc/go1.16#go-command

>  Module-aware mode is enabled by default, regardless of whether a go.mod file is present in the current working directory or a parent directory. More precisely, the GO111MODULE environment variable now defaults to on.